CMS-Enterprise/sbom-harbor
# fix(ui): correct user search url [ISPGCASP-1046]

## Summary

This PR fixes the URL used when searching for a user email to add to a team.

### Fixed

- the incorrect URL was `xxx.cloudformation.net/api/user?filter=...`
- the corrected URL is `xxx.cloudformation.net/api/v1/user/search?filter=...`

## How to test

- `yarn start`
- login, view a team, then edit that team
- type `qui` in the user search dropdown
- observe that the search requests succeed and the results show in the dropdown

## Screenshots

- <img width="857" alt="image" src="https://user-images.githubusercontent.com/1915925/210604386-12b4572c-b8ef-44ae-acea-188effb984c2.png">
